### DISCUSSION TITLE
Allow selectively disabling resource controls on index view.

### DISCUSSION THREAD

**Discussion by MrJoy** [avo-hq/avo#999] [Feature requests]
Currently the only way to do this is via permissions, but it would be nice if we could selectively disable these _generally_ for a given model.

In particular, if I'm using `link_to_resource`, then a button for `show` is just wasted space.

**Comment by adrianthedev** [comment]
I know and I'd like to handle that in a greater refactor where you could more easily customize the actions shown on each row. Maybe even collapse the buttons under a single button somehow.
